static/js/components/Source.jsx

```diff
@@ -73,6 +137,7 @@ const Source = ({ route }) => {
         </div>
 
         <br />
+        {showClassification()}
```

Comment:
See comment above about making this a regular React component that can be rendered in JSX tags

Reply:
Can we leave this for another PR?

Reply:
As someone who will have to edit/update/maintain this code, I'd rather we get it right before it gets merged so that it doesn't fall through the cracks and potentially cause headaches later.

Reply:
done